INSTALL.md

```diff
@@ -161,9 +161,8 @@ $(venv) python3 saskatoon/manage.py createsuperuser --skip-checks
 
 For Django to serve static files during development `SASKATOON_DEBUG` must be set to `yes`.
 
-WARNING: This is not suitable for production use!
+WARNING: This is not suitable for production use! (See [issue 86](https://github.com/LesFruitsDefendus/saskatoon-ng/issues/86))
```

Comment:
To be clear I only meant setting DEBUG=True is not suitable for production. Letting django serve static files in development is not inherently "bad". In production I assume static files are handled by nginx. The main advantage I see of using whitenoise is to get a similar setup in both development and production.

Reply:
Hi, just I agree with this change and yes, I always manage static through nginx, e.g

```
    location /static/ {
        alias /var/www/html/saskatoon-ng/saskatoon/sitebase/static/;
    }

```
Bests,